src/components/Firestore/index.scss

```diff
@@ -24,6 +24,7 @@
 .Firestore-panels {
   display: flex;
   flex-direction: row !important;
+  height: 400px;
```

Comment:
How would this look on taller screens? Can we use `vh`?

Reply:
And smaller screens :)

Reply:
For now I have done 300px min with 55vh height. this seems to fill the screen pretty well. We can tweak in another PR